docker-compose.yml

```diff
@@ -1,75 +1,60 @@
 wordpress:
-  build: .
-  privileged: true
-  env_file: _env
-  environment:
-    - CONTAINERBUDDY=file:///etc/containerbuddy.json
```

Comment:
the Dockerfile already has `ENV CONTAINERPILOT file:///etc/containerpilot.json` in there, so I guess this `envrionment` block was uneeded for a while now.